src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/VulnerabilityLifetimeScore.java

```diff
@@ -16,37 +15,67 @@
 import com.sap.sgs.phosphor.fosstars.model.value.Vulnerability;
 import java.io.IOException;
 import java.io.InputStream;
+import java.time.Duration;
+import java.time.Instant;
 import java.util.Date;
+import java.util.LinkedHashMap;
 import java.util.List;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
+import java.util.Map;
 
 /**
  * The score shows how fast vulnerabilities are patched in an open-source project.
  * The score is based on the following features:
  * <ul>
  *   <li>Information about vulnerabilities in the project.</li>
  *   <li>When the project started.</li>
- *   <li>When the first commit was done.</li>
  * </ul>
  */
 public class VulnerabilityLifetimeScore extends FeatureBasedScore {
 
-  private static final Logger LOGGER = LogManager.getLogger(VulnerabilityLifetimeScore.class);
+  /**
+   * If unknown, the score assumes that the project started 10 years ago.
+   */
+  private static final Date DEFAULT_PROJECT_START_DATE
```

Comment:
Hmm , any specific reason for not using
    now = Calendar.getInstance();
    now.add(Calendar.YEAR, -10);
?

Reply:
Unfortunately, your code doesn't really work. I guess, you meant something like the following:

```java
Date.from(Instant.now().minus(10, ChronoUnit.YEARS))
```

Reply:
In fact, my code doesn't work either. Here is what works:

```java
Date.from(ZonedDateTime.now().minusYears(10).toInstant())
```

Reply:
Good suggestion to use date/time API. I've updated the class.

Reply:
He he ok :) But, do you like to keep the actual calculation one with numbers or this? )

Reply:
I am not sure if I got your question. I've updated the class to use  `Date.from(ZonedDateTime.now().minusYears(10).toInstant())`. Did I miss anything?